grid.cpp

```diff
@@ -153,6 +153,7 @@ void initializeGrids(
       .initialize(comm)
       .set_geometry(geom_params);
 
+   mpiGrid.set_partitioning_option("RCB_RECTILINEAR_BLOCKS", "1");
```

Comment:
Though this proved necessary for eVlasiator, let's rather put it in configs and merge also Leo's "unlimited Zoltan parameters input" branch here.